src/poliastro/core/elements.py

```diff
@@ -220,10 +219,13 @@ def coe2mee(p, ecc, inc, raan, argp, nu):
         \end{align}
 
     """
+    if inc == np.pi:
+        raise ValueError(
+            "Cannot compute the equinoctial parameters, `h` and `k`, due to singularity"
```

Comment:
We must inform user what parameter caused the singularity. What about the following:

```python
raise ValueError(
    "Cannot compute modified equinoctial set for 180 degrees orbit inclination due to `h` and `k` singularity."
)
```

Reply:
Thanks! I used your suggestion verbatim.